### ISSUE TITLE
Invalid elasticsearch(-oss) alternative dependency format breaks package installation/update for RPM based installations.

### ISSUE THREAD

**Issue by jannheider** [zammad/zammad#3470]
### Infos:

* Used Zammad version: 3.6
* Installation method (source, package, ..): package
* Operating system: CentOS Linux release 7.9
* Database + version: postgres (PostgreSQL) 9.2.24
* Elasticsearch version: elasticsearch-7.12.0-1.x86_64
* Browser + version: Any

### Expected behavior:

* Update Zammad with via package to version 4.0 with `yum update` without any dependency problems 

### Actual behavior:

* `yum update` is reporting dependency problems:
```
---> Paket zammad.x86_64 0:3.6.0-1614852413.9a3a289a.centos7 markiert, um aktualisiert zu werden
---> Paket zammad.x86_64 0:4.0.0-1616652753.1200cd9d.centos7 markiert, um eine Aktualisierung zu werden
--> Abhängigkeit elasticsearch|elasticsearch-oss wird für Paket zammad-4.0.0-1616652753.1200cd9d.centos7.x86_64 verarbeitet
--> Abhängigkeitsauflösung beendet
Fehler: Paket: zammad-4.0.0-1616652753.1200cd9d.centos7.x86_64 (zammad)
            Benötigt: elasticsearch|elasticsearch-oss
 Sie können versuchen, mit --skip-broken das Problem zu umgehen.
```

Elasticsearch ist installed up and running in version elasticsearch-7.12

### Steps to reproduce the behavior:

* `yum update`

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by MrGeneration** [comment]
Side note: This has also been discussed in the community https://community.zammad.org/t/update-on-centos-8/6451

**Comment by thorsteneckel** [comment]
Checking possible fix with https://github.com/zammad/zammad/tree/debug-issue_3470 / https://github.com/zammad/zammad/commit/a849ab3867e077ee0bee75da13689d04634e5f9a once packager.io build is ready.

**Comment by mafn** [comment]
Just checked, boolean dependencies were [introduced in rpm 4.13](https://rpm.org/user_doc/boolean_dependencies.html) but CentOS 7 only has 4.11, so the fix (c5560384)
```
- (elasticsearch or elasticsearch-oss)
```
will only work for CentOS>7

**Comment by thorsteneckel** [comment]
You are right. Actually packager.io / fpm / rpmbuild doesn't support the format. We will roll back the change to `elasticsearch` for RPM based package managers for now. Fix is in the making.

**Comment by YosefAdPro** [comment]
@thorsteneckel On CentOS 7 still broken.

**Comment by MrGeneration** [comment]
@YosefAdPro because the built is failing:
https://packager.io/gh/zammad/zammad/refs/stable

Please wait until packager.io built the packages accordingly.

**Comment by thorsteneckel** [comment]
Sorry this issue was closed automatically with my commit.

There is another issue: A dependency we (indirectly use) was yanked today and causes some more changes. We need to work things out. I keep you updated.
EDIT: https://www.theregister.com/2021/03/25/ruby_rails_code/
We're one of those projects.

Reference image:
![image](https://user-images.githubusercontent.com/3873515/112500590-cda2cf80-8d88-11eb-8c4c-613c903f42e2.png)

**Comment by thorsteneckel** [comment]
Everything should be resolved in about 30 minutes from now once the build pipeline for each change has passed: https://packager.io/gh/zammad/zammad/refs/stable

Docker images will get fixed tomorrow.
